microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -46,13 +95,13 @@ If you have a developer account registered on Partner Center, use the correspond
           > After enrollment, the value of this field is read-only.  
           
        :::column-end:::
-    :::row-end:::  
+    :::row-end:::
```

Comment:
This table was not showing up correctly in the preview document.

Reply:
![image](https://user-images.githubusercontent.com/54015428/131387361-8d305a05-c102-4b71-bb99-0a0a99963d3c.png)

Reply:
Here is the preview view to use:
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/create-dev-account?branch=pr-en-us-1398
Does this rendered article look ok on your machine?
I put links to the 3 rendered articles at top of the Conversation page of the PR, including the above URL.

You apparently used the rough Markdown preview feature of GitHub.com, at URL:
https://github.com/MicrosoftDocs/edge-developer/blob/a108b9b61633c68d996578f46a27c07b5a4756a4/microsoft-edge/extensions-chromium/publish/create-dev-account.md
which (by design, I believe) doesn't resolve triple-colon image tagging.